src/components/templates/PosterPage/components/PosterPageSettingsControl/PosterPageSettingsControl.tsx

```diff
@@ -12,59 +12,44 @@ import "./PosterPageSettingsControl.scss";
 
 export interface PosterPageSettingsControlProps {
   venueId: string;
-  isPosterLive?: boolean;
+  isPosterLive: boolean;
 }
 
 export const PosterPageSettingsControl: React.FC<PosterPageSettingsControlProps> = ({
-  isPosterLive = false,
   venueId,
-}) => (
-  <OverlayTrigger
-    trigger="click"
-    placement="bottom-end"
-    overlay={
-      <PosterPageSettingsControlOverlay
-        isPosterLive={isPosterLive}
-        venueId={venueId}
-      />
-    }
-    rootClose={true}
-  >
-    <PosterPageControl label="Settings" icon={faCog} />
-  </OverlayTrigger>
-);
-
-export interface PosterPageSettingsControlOverlayProps {
-  isPosterLive?: boolean;
-  venueId: string;
-}
-
-export const PosterPageSettingsControlOverlay: React.FC<PosterPageSettingsControlProps> = ({
   isPosterLive,
-  venueId,
 }) => {
   const setVenueLiveOn = useCallback(() => {
-    setVenueLiveStatus(venueId, true);
+    setVenueLiveStatus({ venueId, isLive: true });
   }, [venueId]);
 
   const setVenueLiveOff = useCallback(() => {
-    setVenueLiveStatus(venueId, false);
+    setVenueLiveStatus({ venueId, isLive: false });
   }, [venueId]);
 
   return (
-    <Popover
-      id="poster-page-settings-popover"
-      className="PosterPageSettingsControl__popover"
+    <OverlayTrigger
+      trigger="click"
+      placement="bottom-end"
+      overlay={
+        <Popover
```

Comment:
As called out in the previous thread about this, according to the docs/code, it sounds like this should still have a `ref` passed to it when used in this way. And given that, I don't know that it made any sense to revert the separation of the inner content like you've done here.

I've described this in more detail in the previous thread + provided more detail about how/why using `forwardRef` should work in this situation.